workshop_app/forms.py

```diff
@@ -170,48 +91,47 @@ def save(self):
         new_profile = Profile(user=new_user)
         new_profile.institute = cleaned_data["institute"]
         new_profile.department = cleaned_data["department"]
-        #new_profile.position = cleaned_data["position"]
+        # new_profile.position = cleaned_data["position"]
         new_profile.phone_number = cleaned_data["phone_number"]
         new_profile.location = cleaned_data["location"]
         new_profile.title = cleaned_data["title"]
         new_profile.state = cleaned_data["state"]
-        new_profile.how_did_you_hear_about_us = cleaned_data["how_did_you_hear_about_us" ]
+        new_profile.how_did_you_hear_about_us = cleaned_data["how_did_you_hear_about_us"]
         new_profile.activation_key = generate_activation_key(new_user.username)
-        new_profile.key_expiry_time = timezone.now() + \
-                                    timezone.timedelta(days=1)
+        new_profile.key_expiry_time = timezone.now() + timezone.timedelta(days=1)
         new_profile.save()
         key = Profile.objects.get(user=new_user).activation_key
```

Comment:
No need of doing this query again. Store the result obtained from `generate_activation_key` function and use the same wherever necessary